docs/develop/convert-javascript-to-typescript.md

```diff
@@ -79,12 +77,28 @@ This article shows you how to create an Excel add-in using Visual Studio and the
 	declare var fabric: any;
 	```
 
-12. In the **Home.ts** file, change **'1.1'** to **1.1** (that is, remove the quotation marks) in the following line, and save the file:
+12. In the **Home.ts** file, change **'1.1'** to **1.1** (that is, remove the quotation marks) in the following line:
 
 	```javascript
 	if (!Office.context.requirements.isSetSupported('ExcelApi', '1.1')) {
 	```
 
+13. In the **Home.ts** file, find the `displaySelectedCells` function, replace the entire function with the following code, and save the file:
+
+    ```javascript
```

Comment:
Shouldn't this be TypeScript?

Reply:
Yes!  Good catch; I'll update this and the other 2 snippets from Home.ts.